styles.css

```diff
@@ -11,6 +11,11 @@ body {
   padding-top: 0px;
 }
 
+.main-info-container-fluid {
+  margin-left: 172px;
+  margin-right: 172px;
```

Comment:
Hey Iman, sorry to get this to you so late but we were looking for 342px on each side, not 342/2px. If you could update this we'll approve right away. 


Thank you!